base/operators.jl

```diff
@@ -944,6 +944,9 @@ Function composition also works in prefix form: `∘(f, g)` is the same as `f 
 The prefix form supports composition of multiple functions: `∘(f, g, h) = f ∘ g ∘ h`
 and splatting `∘(fs...)` for composing an iterable collection of functions.
 
+In the splatting of a composed function, say `fs[...]`, functions are called in
+descending order, i.e. `func in fs[end:-1:begin]`, till the last function is called.
```

Comment:
I had to read this a few times. 

`fs[...]` seems confusing, is `fs` the type, what is this? 

I think it's important to convey that this is not a new property under splatting being defined. It's just a note about what follows from properties already described. On the first line, "`(f ∘ g)(args...; kwargs...)` means `f(g(args...; kwargs...))`" means that `f` is executed last.

Perhaps it can be integrated into the example of the previous paragraph, instead of introducing another array of functions. 
```suggestion
The prefix form supports composition of multiple functions: `∘(f, g, h) == f ∘ g ∘ h`.
When splatting an iterable collection of functions `∘(fs...)`, notice that `first(fs)` is executed last:
`fs = [f, g, h]` gives `∘(fs...)(x) ==  (f ∘ g ∘ h)(x) == f(g(h(x)))`.
```

Reply:
> I had to read this a few times.
> 
> `fs[...]` seems confusing, is `fs` the type, what is this?
> 
> I think it's important to convey that this is not a new property under splatting being defined. It's just a note about what follows from properties already described. On the first line, "`(f ∘ g)(args...; kwargs...)` means `f(g(args...; kwargs...))`" means that `f` is executed last.
> 
> Perhaps it can be integrated into the example of the previous paragraph, instead of introducing another array of functions.

I think your suggestion is very neat and great, but then it has abstracted the whole concept and for any user who has not come across the function `first`, this doesn't do him/her any good.

I explicitly provided `fs[end:-1:begin]` to kinda may it a lay term for anyone to quickly pickup.

Anyways I'd change `fs[...]` to `∘(fs...)` though.

Reply:
If you want to avoid `first`, you could write say "notice that `fs[end]` is executed first:", which seems as clear.

Reply:
> If you want to avoid `first`, you could write say "notice that `fs[end]` is executed first:", which seems as clear.
> 
> I was keen to avoid `fs[end:-1:begin]`, as that means there are 3 different orderings to think about.

Yes `fs[end]` is very concise.